src/extensions/cluster-feature.ts

```diff
@@ -39,96 +39,96 @@ export abstract class ClusterFeature {
     canUpgrade: false
   };
 
+  constructor(protected cluster: KubernetesCluster){}
```

Comment:
Are we tracking breaking changes?

Reply:
We should be, @jakolehm is going to be change this even more (I think eventually removing this class all together). We should have a doc somewhere for keeping track of all the breaking changes.